Makefile

```diff
@@ -156,7 +172,7 @@ endif
 test: pytest
 
 pytest:
-	$(SHOW)MODULE=$(abspath $(TARGET)) ./tests/pytest/tests.sh
+	$(SHOW)MODULE=$(abspath $(TARGET)) $(realpath ./tests/pytest/tests.sh)
```

Comment:
realpath (sadly) still isn't completely portable. Can you use realink -f?

Reply:
This in gnu make's `realpath`, not `/bin/realpath`. This fortunately works properly.

Reply:
BTW the reason we require `realpath` here in the first place is because using a relative path causing `test.sh` not to print program name properly when it detects an error. So this needs to be fixed to (probably something like `PROGNAME="$0"` at the top).